shepherd/Utils.py

```diff
@@ -0,0 +1,42 @@
+from enum import Enum, unique
+
+class Header(Enum):
+    # Used to autoenumerate
```

Comment:
Seems like this should be described with a doc string, but it really doesn't matter.